src/components/Hero/Hero.tsx

```diff
@@ -68,35 +68,37 @@ export default function Hero(props: React.PropsWithChildren<HeroProps>) {
   if (heroType === HeroTypes.Primary) {
     if (!backgroundImageSrc) {
       console.warn(
-        `Warning: it is recommended to use the "backgroundImageSrc" prop for PRIMARY hero.`
+        `NYPL Reservoir Hero: it is recommended to use the "backgroundImageSrc" prop for PRIMARY hero.`
```

Comment:
`PRIMARY` should not be capitalized, but I am not exactly sure what it should be replaced with.  `HeroTypes.Primary`?  And this change is applicable to all references to the `Hero` types.